span-normalizer/span-normalizer-api/src/main/avro/trace-identity/PeerIdentity.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("org.hypertrace.core.spannormalizer")
```

Comment:
This requires in raw-spans-grouper. Shouldn't this be part of `raw-spans-grouper`

Reply:
There were some avros also defined here so introduced them in a common place. Can move them raw-spans-grouper